src/app/test/test.component.ts

```diff
@@ -223,6 +225,13 @@ export class TestComponent implements OnInit {
     }
   }
 
+  deleteAllReports(): void {
+    this.httpService.deleteAllReports(this.testReportsService.storageName).subscribe({
+      next: () => this.testReportsService.getReports(),
+      error: () => catchError(this.errorHandler.handleError()),
+    });
+  }
```

Comment:
This should have the new way of error checking as well.

Reply:
Added the change